crates/estree/src/expr.rs

```diff
@@ -171,3 +171,302 @@ impl IntoExpr for Object {
         })
     }
 }
+
+impl<'a> Serialize for Serialization<'a, ExprListItem> {
+    fn serialize<S: Serializer>(&self, serializer: S) -> ::std::result::Result<S::Ok, S::Error> {
+        use easter::expr::ExprListItem::*;
+        match *self.data() {
+            Expr(ref expr) => Serialization::new(expr).serialize(serializer),
+            Spread(_, ref expr) =>
+                tag(json!({
+                    "type": "SpreadElement",
+                    "argument": Serialization::new(expr)
+                })).serialize(serializer)
+        }
+    }
+}
+
+impl<'a> Serialize for Serialization<'a, Op<UnopTag>> {
+    fn serialize<S: Serializer>(&self, serializer: S) -> ::std::result::Result<S::Ok, S::Error> {
+        use easter::punc::UnopTag::*;
+        match self.data().tag {
+            Minus   => "-",
+            Plus    => "+",
+            Not     => "!",
+            BitNot  => "~",
+            Typeof  => "typeof",
+            Void    => "void",
+            Delete  => "delete"
+        }.serialize(serializer)
+    }
+}
+
+impl<'a> Serialize for Serialization<'a, Op<BinopTag>> {
+    fn serialize<S: Serializer>(&self, serializer: S) -> ::std::result::Result<S::Ok, S::Error> {
+        use easter::punc::BinopTag::*;
+        match self.data().tag {
```

Comment:
Maybe best to put string->operator conversions as part of operators themselves so that they could be potentially reused for other serialization targets?